build/angular-slider.js

```diff
@@ -1395,7 +1410,10 @@ angular.module('vr.directives.slider', ['ngTouch']).directive('slider',
                                  */
                                 function redrawSliders() {
                                     setPointers();
-                                    adjustBubbles();
+                                    // Only adjust bubbles if allowed
+                                    if(scope.noBubbleAdjustment === undefined) {
```

Comment:
Space after `if` (it was greg's style which is inconsistent with our style guide)